cmd/dashboard/repl.js

```diff
@@ -28,6 +34,14 @@ class REPL {
       writer: this.enhancedWriter.bind(this)
     });
 
+    if (fs.existsSync(this.cmdHistoryFile)) {
+      fs.readFileSync(this.cmdHistoryFile)
+        .toString()
+        .split('\n')
+        .reverse()
+        .forEach((cmd) => { this.replServer.history.push(cmd); })
+    }
```

Comment:
We should probably use an event to ask the command history module for the command history instead of reading it here.

Reply:
@jrainville I implemented event-based history for repl and forgot to push during rebase (actually wipe was caused by reset --hard HEAD).... I will have to re-implement this tonight.